releasenotes/notes/add-image_id-field-to-ad-tags-331636a9c11a2131.yaml

```diff
@@ -0,0 +1,5 @@
+
+---
+upgrade:
+  - |
+    Tags in activity dumps now have an additional field called "image_id". It is reported as such: "repo+@+sha256:XXX"
```

Comment:
It would be great to mention CWS in here, do we even need one @lebauce ?

Reply:
I don't think we need a release note for this